packages/react-icons/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@tonic-ui/react-icons",
+  "version": "1.37.0",
+  "description": "React Tonic UI Icons library.",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "generate-icons": "node scripts/generate-icons.mjs",
+    "build": "yarn generate-icons && cross-env PACKAGE_NAME=index INPUT=src CJS_OUTPUT_DIRECTORY=dist/cjs ESM_OUTPUT_DIRECTORY=dist/esm rollup --config rollup.config.mjs",
+    "clean": "del build coverage dist",
+    "lint": "eslint --ext .js,.jsx,.mjs .",
+    "pre-push": "bash -c 'echo -e \"=> \\e[1;33m$npm_package_name\\e[0m\"' && yarn run build && yarn run lint && yarn run test",
+    "prepublish": "yarn run build",
+    "test": "exit 0"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@tonic-ui/react": "^1.29.0"
+  },
+  "devDependencies": {
+    "@babel/plugin-transform-runtime": "^7.24.0",
+    "@babel/preset-env": "^7.24.0",
+    "@babel/preset-react": "^7.23.3",
+    "@emotion/babel-plugin": "^11.11.0",
+    "@trendmicro/babel-config": "^1.0.2",
+    "@trendmicro/tmicon": "1.x",
```

Comment:
Thank you for refining/fixing some of my changes. I had a question about the versioning of `tmicon` in `react-icons`. Since it looks like we are pinning the `react-icons` package version to `tmicon` latest `1.37.0`, would it also make sense to pin the version to `1.37.0` for the `tmicon` dependency in `react-icons`'s `package.json`?

Reply:
@ZachLegros You're correct. It should pin the version of the `tmicon` dependency to 1.37.0 to ensure consistency.